exam/models/exam.py

```diff
@@ -27,15 +14,34 @@ class StudentStudent(models.Model):
                                        'Exam History', readonly=True)
 
     @api.model
-    def search(self, args, offset=0, limit=None, order=None, count=False):
+    def name_search(self, name='', args=None, operator='ilike',
+                    limit=100):
```

Comment:
because,in v10 def search method is not working ,so we are used the name_search but now we check the code of the models.py file that it is name chnages  to def _search method .

So,we have the changes method name_search to _search